src/components v2/Atoms/Components/InfoCard/InfoCard.spec.js

```diff
@@ -0,0 +1,94 @@
+import InfoCard from '@/components v2/Atoms/components/InfoCard/InfoCard.vue';
```

Comment:
This line fails in the test suite.

Reply:
ah yes, so I changed it to a relative path and it now passes